Jellyfin.Server.Implementations/Security/AuthorizationContext.cs

```diff
@@ -219,24 +216,7 @@ await using (dbContext.ConfigureAwait(false))
         /// <summary>
         /// Gets the auth.
         /// </summary>
-        /// <param name="httpReq">The HTTP req.</param>
-        /// <returns>Dictionary{System.StringSystem.String}.</returns>
-        private static Dictionary<string, string>? GetAuthorizationDictionary(HttpContext httpReq)
-        {
-            var auth = httpReq.Request.Headers["X-Emby-Authorization"];
```

Comment:
Removing support for this header will cause issues with at least:

- The C# SDK
- The Kodi client
- Jellycon
- The mopidy client/thing

And potentially more clients that changed the header but didn't release yet and third-party clients not includes in my [search](https://github.com/search?q=org%3Ajellyfin%20X-Emby-Authorization&type=code).

Reply:
CC @mcarlton00 @crobibero
See [this gist](https://gist.github.com/nielsvanvelzen/ea047d9028f676185832e51ffaf12a6f) for documentation on API authorization

Reply:
This doesn't remove support for `X-Emby-Authorization`, it just removes this duplicate function.

Reply:
Thanks for clarifying, the diff made it look like the header was completely removed.